### ISSUE TITLE
Spotbugs-4.0.0 doesn't detect FLOATING_POINT_EQUALITY

### ISSUE THREAD

**Issue by ChristopherNeufeld** [spotbugs/spotbugs#1097]
I have seen a change in behaviour, using the test cases in spotbugs.
Downloaded the binary packages for spotbugs-3.1.12 and spotbugs-4.0.0.
In the source package, there is a file, spotbugs-4.0.0/spotbugsTestCases/src/java/FloatingPointEquality.java.
Compiled this file: javac FloatingPointEquality.java
My javac is version 1.8.0_232 on Linux.
I then ran both spotbugs versions against the resulting .class file.  With spotbugs-3.1.12:

```
H D FE: Test for floating point equality in FloatingPointEquality.sum(double)  At FloatingPointEquality.java:[line 5]
Warnings generated: 1
```
With spotbugs-4.0.0, no output is generated.
